lib/string.gi

```diff
@@ -1406,3 +1406,61 @@ InstallGlobalFunction(PrintFormatted, function(args...)
     # directed
     Print(CallFuncList(StringFormatted, args));
 end);
+
+InstallGlobalFunction(Pluralize,
+function(args...)
+  local nargs, str, len;
+
+  nargs := Length(args);
+  if not (nargs in [2, 3] and
+          IsInt(args[1]) and args[1] >= 0 and
+          ForAll([2 .. nargs], i -> IsString(args[i]))) then
+    ErrorNoReturn("Usage: Pluralize(<count>, <string>[, <plural>])");
+  elif args[1] = 1 then
+    return args[2];
+  elif nargs = 3 then
+    return args[3];
+  fi;
+  str := args[2];
+  len := Length(str);
+
+  # Special case weirdly short words
+  if len <= 1 then
```

Comment:
actually, how many two letter nouns are there? perhaps this case could also be excluded

Reply:
Yeah. I'll change things around. I've had a look and can only find the two letter words "ax" and "ex" as ones which I would pluralise differently than just adding 'x'. So I'll move the "ending in x" rule earlier, and then special case `len = 2` as well.

Reply:
Actually I'll ignore "ax" and "ex". I don't think they'll be used (not that `Pluralize` itself will be used much!), and if they are, people will just have to use the 3-argument version, or modify `Pluralize`.